tsl/src/compression/create.c

```diff
@@ -867,6 +868,7 @@ check_modify_compression_options(Hypertable *ht, WithClauseResult *with_clause_o
 	 * empty list. */
```

Comment:
You should probably update this comment, which is not entirely accurate any more.